src/main/java/org/team1540/robot2019/commands/hatch/ExtendHatch.java

```diff
@@ -0,0 +1,16 @@
+package org.team1540.robot2019.commands.hatch;
+
+import edu.wpi.first.wpilibj.command.InstantCommand;
+import org.team1540.robot2019.Robot;
+
+public class ExtendHatch extends InstantCommand {
```

Comment:
Should we maybe have variants, some being instant (for, say, retracting the mech before floor intaking a cargo) and some waiting for a small amount of time (for, say, auto placement?)